cli/README.md

```diff
@@ -1,55 +1,74 @@
-# Xata CLI
+# Command Line Interface (CLI)
 
-## Installing
+Xata provides a CLI as a convenience for developers. It is intended to provide feature parity with Xata's [Web UI](https://docs.xata.io/getting-started) (User Interface) to enable developers to build applications at **rapid pace** with **as little friction as possible**.
 
-The recommended way of installing the CLI is by installing it as an npm package globally:
+## Installation
 
-```bash
-npm install @xata.io/cli -g
+Our CLI is distributed both as an **[npm](https://www.npmjs.com/package/@xata.io/cli) package** for developers who work with Node.js, and a **cross-platform binary** for those who don't. Below, you'll find commands that you can copy and paste into your platform's shell that will help you install the CLI on your specific platform.
+
+![Installing the Xata CLI](/screenshots/CliInstall.gif)
+
+### Installing with npm
+
+To install the Xata CLI globally, run the following command:
+
+```sh
+npm i --location global @xata.io/cli
 ```
 
-Now you can just run `xata --help` to see all the available commands.
+This will install the Xata CLI. We recommend installing it globally because it becomes much more convenient to work with, having you invoke a `xata` command instead of the more convoluted alternative with a project-scoped installation (`npm run xata`), or an npx-based installation (`npx @xata.io/cli`). The alternative approaches also work depending on your preference.
+
+### Installing the Binary without npm
+
+Our CLI can also be installed as a cross-platform binary for our users who do not have `npm` installed on their systems. Depending on your platform of choice, here are installation instructions to get you started.
```

Comment:
To be sure, this is not yet available. The below instructions are currently installing the legacy Go CLI.

I was thinking to remove or hide this section until we add back the binary install option, otherwise we're going to get trouble with people installing the old CLI.

Reply:
Sorry, I missed this: https://xata-hq.slack.com/archives/C032EKCBUGM/p1656916087379719?thread_ts=1656873846.216119&cid=C032EKCBUGM

Let's remove it. 💪

Reply:
Done in https://github.com/xataio/client-ts/pull/400/commits/ffc85ef36adff7dab19618f707613e1d01adb70a